LibreNMS/OS/Shared/Cisco.php

```diff
@@ -405,8 +405,8 @@ public function pollNac()
                     'timeout' => $portAuthSessionEntryParameters['cafSessionTimeout'],
                     'time_left' => $portAuthSessionEntryParameters['cafSessionTimeLeft'],
                     'vlan' => $portAuthSessionEntryParameters['cafSessionAuthVlan'],
-                    'authc_status' => $session_info['authc_status'],
-                    'method' => $session_info['method'],
+                    'authc_status' => $session_info['authc_status'] ?: "", //adds null
```

Comment:
These should be `$session_info['authc_status'] ?? ''`
(this is the nullable operator, no comment is necessary)